spec/features/create_items.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper.rb'
```

Comment:
`require 'rails_helper.rb'` can be shortened to `require 'rails_helper'`.

The `require` method automatically adds the `.rb` extension.